mbuild/utils/geometry.py

```diff
@@ -55,3 +55,27 @@ def coord_shift(xyz, box):
         xyz += box_max
 
     return xyz
+
+def wrap_coords(xyz, box):
+    """ Wrap coordinates inside box
+
+    Parameters
+    ---------
+    xyz : numpy.array of points with shape N x 3
+    box : numpy.array specifing the size of box ie [Lx, Ly, Lz]
+
+    Returns
+    -------
+    wrap_xyz : numpy.array of points with shape N x 3
+
+    Notes
+    -----
+    Assumes we are wrapping inside the positive octant
+    Currently only supports orthorhombic boxes
+    """
+    box_arr = np.asarray(box)
```

Comment:
One last nit - it looks like this only works on array-likes representing boxes of `[0, 0, 0, Lz, Ly, Lz]`. Is this meant to only handle that case, or should we be able to wrap it into `[xmin, ymin, zmin, xmax, ymax, zmax]` type boxes? Also it might be worth considering sanitizing an input of a `mbuild.Box`, which could either consider the min/max case or just operate on the `.lengths`.

Reply:
Okay, I lied - I have another final suggestion. Right now it takes in a list-like, which is fine, but I would also like it to take in an `mb.Box`. i.e. I can do this

```
xyz = mb.utils.geometry.wrap_coords(filled.xyz, small_box.lengths)
```

but I would like to be able to pass it just the box and let it do the wrapping on that, ideally also considering non-zero mins.